src/util/Options.js

```diff
@@ -69,6 +69,9 @@
  * [User Agent](https://discord.com/developers/docs/reference#user-agent) header
  * @property {PresenceData} [presence={}] Presence data to use upon login
  * @property {IntentsResolvable} intents Intents to enable for this connection
+ * @property {number} [fetchGuildTimeout=15000] Time in milliseconds that Clients with the GUILDS intent should hang
+ * and wait for missing guilds to be recieved before starting the bot.
+ * If not specified, the default is 15 seconds.
```

Comment:
```suggestion
 * @property {number} [fetchGuildTimeout=15000] Time in milliseconds to wait for missing guilds to be received before marking as ready (0 to skip this timeout)
```

Reply:
The only good thing about this change is that it mentions that 0 skips the timeout. Other than that, why??
You removed "Clients with the GUILDS intent" and "If not specified, the default is 15 seconds" which imo are quite important mentions, what's the purpose of this?